plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -475,4 +485,34 @@ public DeltaLakeConfig setRegisterTableProcedureEnabled(boolean registerTablePro
         this.registerTableProcedureEnabled = registerTableProcedureEnabled;
         return this;
     }
+
+    @Min(value = MIN_READER_VERSION, message = "Must be in between " + MIN_READER_VERSION + " and " + MAX_READER_VERSION)
+    @Max(value = MAX_READER_VERSION, message = "Must be in between " + MIN_READER_VERSION + " and " + MAX_READER_VERSION)
+    public int getDefaultReaderVersion()
+    {
+        return defaultReaderVersion;
+    }
+
+    @Config("delta.default-reader-version")
+    @ConfigDescription("The reader version used by tables created by Trino")
```

Comment:
"by Trino" is a little redundant I think :)
```suggestion
    @ConfigDescription("The default reader version used by new tables")
```